public/contribute/_overview.md

```diff
@@ -20,12 +20,11 @@ Be sure to follow the [contribution guidelines](/contribute/contributing-code.ht
 
 ## Testing
 
-For a quick overview of the overall test results for Crosswalk
-binaries, see
-[Crosswalk test result indicators](https://crosswalk-project.org/#wiki/Crosswalk-test-result-indicators).
+For a quick overview of the overall test results for Crosswalk binaries, see [Crosswalk quality dashboard](/documentation/quality_dashboard.html).
 
```

Comment:
The link does not exist:  /documentation/quality_dashboard.html    Did you forget to check in this new file?